src/scribe_data/extract_transform/languages/Japanese/nouns/query_nouns.sparql

```diff
@@ -1,11 +1,12 @@
+PREFIX wikibase: <http://wikiba.se/ontology#>
```

Comment:
Checking in on this, @henrikth93 :) I've seen this in other queries, but don't do it normally myself. What's the purpose of it?